backend/src/api/cards.py

```diff
@@ -62,6 +63,50 @@ def get_cards(
     return [CardOut.from_card(card) for card in cards]
 
 
+@router.get("/random", response_model=List[CardOut])
+def get_random_cards(
+    deck_ids: str = "",
+    count: int = 10,
```

Comment:
Could we use one of the below to avoid having to manually check `count < 1`?

```
count: int = Field(default=10, gt=0),
...
count: int = Query(default=10, gt=0, description="Number of cards to return"),
```

Field seems to be helpful in more simple cases (like this) while Query gives more control and lets you add description etc that show up in OpenAPI docs. Not sure if we need that.